### ISSUE TITLE
Accumulation of hemosiderin laden macrophages

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5558]
**Preferred term label:**
Accumulation of hemosiderin laden macrophages

**Synonyms**

**Definition (free text, please give PubMed ID)**
Erythrocyte fragments are found in the macrophages, leading to the diagnosis of previous pulmonary hemorrhage.
Costabel U. Atlas of bronchoalveolar lavage. London: Chapman & Hall; 1998. ISBN0412792702.

**Parent term (use hpo.jax.org/app)**
Abnormalities of alveolar macrophages

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
@GrieseAG
@Katrinknoflach

**Comment by GrieseAG** [comment]
**Diseases characterized by this term?** same as in Pulmonary hemorrhage HP:0040223
ORPHA:520, OMIM:616414, OMIM:603585, ORPHA:183, OMIM:233450, ORPHA:900, 
ORPHA:79124, ORPHA:363611, ORPHA:509, OMIM:222700, ORPHA:2038, OMIM:178550, ORPHA:238459, ORPHA:906,

**Comment by GrieseAG** [comment]
Definition: 
Hemosiderin-laden macrophages (HLM) in bronchoalveolar lavage (BAL) fluid were originally known as adiagnostic biomarker of alveolar hemorrhage, but have also been observed in idiopathic pulmonary fibrosis (IPF) with histopathological pattern of usual interstitial pneumonia (UIP). 

Comment: Hemosiderin, a product of hemoglobin degradation, appears at least 48-72 hours after bleeding and is helpful in distinguishing diffuse alveolar hemorrhage from surgical trauma. Mild interstitial thickening, organizing pneumonia, or diffuse alveolar damage can also be seen.

PMID: 23678356, 28166761

**Comment by pnrobinson** [comment]
added as HP:0032979
